sanic_openapi/utils.py

```diff
@@ -49,8 +49,26 @@ def get_blueprinted_routes(app):
             continue
 
         for route in blueprint.routes:
+            if not hasattr(route, "handler"):
+                # in sanic 21.3, "route" is now something different:
+                #   an instance of sanic_routing.route.Route
+                # can't typecheck because code needs to be compatible
+                # with older sanic versions, but "handler" does not exist
+                # on sanic>=21.3 route
+                #
+                # the good news - now class based views are already handled
+                for method_handlers in route.handlers.values():
```

Comment:
I think I broke this part on you.... :grimacing: 

You should not need to do all this crazy nested looking to get the handler:

```
yield blueprint.name, route.handler
```

Reply:
FYI - I am building the patch for this and will push it.